README.md

```diff
@@ -41,13 +41,65 @@ $runner->addExtension(new \mageekguy\atoum\ruler\extension($script));
 
 ## Use it
 
-Examples :
+The extension adds a `--filter` option to atoum. This line now appears on the atoum help :
```

Comment:
You miss the space before `:`. No space before `:` in English, a non-breaking space before `:` in French.